gas-estimation/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "gas-estimation"
+version = "0.1.0"
+authors = ["Gnosis Protocol Team <dex@gnosis.io>"]
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+async-trait = "0.1"
+log = "0.4"
+serde = { version = "1.0", features = ["derive"] }
+serde_with = "1.5"
+web3 = "0.13"
```

Comment:
```suggestion
web3 = { version = "0.13", default-features = false }
```

Reply:
Is there a particular reason for this suggestion? I can only guess that we do not use any of the default features and want to pull in less dependencies.

Reply:
Yes, only depend on what you need is a good practice.